speasy/webservices/amda/_impl.py

```diff
@@ -84,15 +85,46 @@ def update_inventory(self):
         data_tree.amda.Parameters.__dict__.update(data.dataRoot.AMDA.__dict__)
 
         self._update_lists()
+    def parameter_concat(self, param1, param2):
```

Comment:
Use this https://github.com/SciQLop/speasy/blob/main/speasy/products/variable.py#L177 instead to avoid code duplication.

Reply:
`parameter_concat` was removed